common-service-core/src/main/java/com/ca/mfaas/monitoring/LatencyUtilsConfigInitializer.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package com.ca.mfaas.monitoring;
+
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+
+/**
+ * This class initializes the system property that disables LatencyUtils thread that measures time by low level java api
+ * This needs to be done before Spring context starts to initialize
+ *
+ * This class has been
+ */
+public class LatencyUtilsConfigInitializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
```

Comment:
Try using @Nonnull annotation

Reply:
But not `@NotNull`, it's `@Nullable` should be, because, you sending to this method null in test. So, you have two options: create application context object and send to this method, or add this `@Nullable` annotation. I would suggest to create some common context for this test and use it, for example:
`private final ConfigurableApplicationContext ctx = new GenericApplicationContext();`

P.s.: wrong line, the correct is 25 :)